package.json

```diff
@@ -29,7 +29,8 @@
     "lodash": "~4.17.21",
     "nconf": "~0.10.0",
     "objection": "^2.2.15",
-    "pg": "~8.6.0"
+    "pg": "~8.6.0",
+    "ret": "^0.4.2"
```

Comment:
`ret` was missing and seemed to cause issues during `npm install` so I installed it directly to fix it